### PR TITLE
Mar 3468

### PR DESCRIPTION
**Pull request by Reshma-M-A**
I am adding **Organisation** and **Contact Number** for the brief and putting it under Brief Details

 
Screen Shot 2019-09-18 at 3 14 28 PM

### CODE REVIEW

File: app/templates/index.html [hunk 1]
```diff
@@ -67,6 +67,16 @@
                   <input type="submit" value="Search" class="button-save">
               </form>
 
+              <form action="{{ url_for('.find_suppliers') }}" method="get" class="question">
```
Comment (malmas18) [inline app/templates/index.html]: did you need to add this?

File: app/templates/view_buyers.html [hunk 2]
```diff
@@ -78,7 +83,7 @@ <h3>Users</h3>
         {% call summary.field() %}
           {{ item.phoneNumber }}
         {% endcall %}
-
+
```
Comment (malmas18) [inline app/templates/view_buyers.html]:
```suggestion
```

File: app/templates/view_buyers.html [hunk 3]
```diff
@@ -54,6 +54,11 @@
 {% if brief %}
 <div>
 <hr>
+<h3>Brief Details</h3>
+Organisation: {{brief.organisation}}
+<br>
```
Comment (malmas18) [inline app/templates/view_buyers.html]:
```suggestion
<br/>
```

File: app/templates/view_buyers.html [hunk 4]
```diff
@@ -54,6 +54,11 @@
...
+ContactNumber: {{brief.contactNumber}}
+<hr>
```
Comment (malmas18) [inline app/templates/view_buyers.html]:
```suggestion
<hr/>
```

File: app/templates/view_buyers.html [hunk 5]
```diff
@@ -54,6 +54,11 @@
 {% if brief %}
 <div>
 <hr>
```
Comment (malmas18) [inline app/templates/view_buyers.html]:
```suggestion
<hr/>
```

File: app/main/views/buyers.py [hunk 6]
```diff
@@ -71,7 +71,6 @@ def find_buyer_by_brief_id():
 
     try:
         brief = data_api_client.get_brief(brief_id).get('briefs')
-
```
Comment (malmas18) [inline app/main/views/buyers.py]: add the line back

File: app/templates/index.html (hunk 1 above)
Comment (Reshma-M-A) [reply app/templates/index.html]: removed

File: app/main/views/buyers.py (hunk 6 above)
Comment (Reshma-M-A) [reply app/main/views/buyers.py]: line added

**Pull request merged**